src/DynamoCore/Graph/Annotations/AnnotationModel.cs

```diff
@@ -493,7 +493,7 @@ void SerializeCore(XmlElement element, SaveContext context)
             helper.SetAttribute("InitialTop", this.InitialTop);
             helper.SetAttribute("InitialHeight", this.InitialHeight);
             helper.SetAttribute("TextblockHeight", this.TextBlockHeight);
-            helper.SetAttribute("background", (this.Background == null ? "" : this.Background.ToString()));
+            helper.SetAttribute("backgrouund", (this.Background == null ? "" : this.Background.ToString()));
```

Comment:
Did we have this spelling mistake for XML format?

Reply:
yes, it needs to stay, explained on slack.